content/blog/2020/08/2020-08-12-windows-installers-upgrade.adoc

```diff
@@ -0,0 +1,102 @@
+ 5. The JENKINS_HOME directory is placed in the LocalAppData directory for the user that the service will run as, this aligns with modern Windows file system layouts.
+ 6. The installer has been updated with branding to make it look nicer and provide a better user experience. 
+ 
+## Screenshots
+
+You may see below the sequence of screenshots for the new installer:
+
+### Step 1
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_1.png[Installer Startup, role=center]
+
+We can see now the Jenkins logo as a prominent part of the unstaller UI.
+
+### Step 2
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_2.png[Installation Directory, role=center]
+
+Now the Jenkins logo and name are in the header during entire process of installation.
+
+### Step 3
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_3.png[Account Selection, role=center]
+
+Now the installer allows both specifying and testing your credentials by validating that the account has LogonAsService rights.
+
+### Step 4
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_4.png[Port Selection, role=center]
+
+Now the installer also allows specifying the port that Jenkins should run on and will not continue until a valid port is entered and tested.
+
+### Step 5
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_5.png[JRE Selection, role=center]
+
+Now instead of bundling a JRE, the installer searches for a compatible JRE on the system (in the current search no JRE was installed). In case you would like to use a different JRE from the one found by the installer, you can browse and specify it. Only JRE 8 and JRE 11 runtimes are supported. In case the selected JRE is found to be version 11 the installer will
+automatically add the necessary arguments and additional jar files for running under Java 11.
+
+### Step 6
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_6.png[Install It, role=center]
+
+All of the items that users can enter in the installer should be overridable on the command line for automated deployment as well. The full list of properties that can be overridden will be available soon.
+
+## Next Steps
+
+Currently the members of the Platform SIG are reviewing the new installer, but we need people to test the installer and give a feedback. If you are interested in testing the new installer, please join the link:https://gitter.im/jenkinsci/platform-sig[Platform SIG gitter room] for more information.
+
+There are still some things that are being researched and implemented in the new installer (e.g., keeping port and other selections when doing an upgrade), but it is getting close to release.
+
+In addition to updates to the MSI based Windows installer, the Platform SIG is working on taking over the link:https://chocolatey.org/packages/jenkins[Chocolatey Jenkins package] and releasing a version for each update.

```

Comment:
We could offer users choices as they consider their upgrade path
```suggestion
Windows users have alternatives for their existing Jenkins installations:

Upgrade from inside Jenkins::
The "Manage Jenkins" section of the running Jenkins will continue to include an "Upgrade" button for Windows users.
You may continue to use that "Upgrade" button to update the Jenkins installation on your Windows computer.
Upgrade from inside Jenkins will continue to use the current Java version.
Upgrade from inside Jenkins will continue to use the current installation location.

Upgrade with the new Jenkins MSI installer:
If you run the new Jenkins MSI installer on your Jenkins that was installed with the old Jenkins MSI installer, it will ... fill in the blanks ...
After the new Jenkins MSI installer has run, the "Manage Jenkins" section of the running Jenkins will continue to include an "Upgrade" button for Windows users.
You may continue to use that "Upgrade" button to update the Jenkins installation on your Windows computer.
Upgrade from inside Jenkins will continue to use the Java version selected when the new Jenkins MSI installer ran.
Upgrade from inside Jenkins will continue to use the new installation location.

```